manager/manager.go

```diff
@@ -727,3 +526,228 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 	})
 
 }
+
+// handleLeadershipEvent read out and discard all of the messages when the manager is stopped,
```

Comment:
nit: comment says handleLeadershipEvent but function name is handleLeadershipEvents